aws-sdk/aws-sdk.d.ts

```diff
@@ -146,16 +146,23 @@ declare module "aws-sdk" {
 		constructor(options?: any);
 		publish(request: Sns.PublishRequest, callback: (err: any, data: any) => void): void;
 	}
-  
+
 	export class SimpleWorkflow {
 		constructor(options?: any);
 		public client: Swf.Client;
 	}
 
 	export class S3 {
-		constructor(options?: any);
-		putObject(params: s3.PutObjectRequest, callback: (err: any, data: any) => void): void;
-		getObject(params: s3.GetObjectRequest, callback: (err: any, data: any) => void): void;
+        constructor(options?: any);
+        putObject(params: s3.PutObjectRequest, callback: (err: any, data: any) => void): void;
+        getObject(params: s3.GetObjectRequest, callback: (err: any, data: any) => void): void;
+        putObject(params: s3.PutObjectRequest, callback: (err: Error, data: any) => void): void;
```

Comment:
This overload is unobservable because the one two lines above is always chosen instead.